modules/caddyhttp/reverseproxy/httptransport.go

```diff
@@ -71,6 +72,22 @@ type HTTPTransport struct {
 	// connecting to an upstream. Default: off.
 	ProxyProtocol string `json:"proxy_protocol,omitempty"`
 
+	// URL to the server that the HTTP transport will use to proxy
+	// requests to the upstream. See http.Transport.Proxy for
+	// information regarding supported protocols. This value takes
+	// precedence over `HTTP_PROXY`, etc.
+	//
+	// Providing a value to this parameter results in
+	// requests flowing through the reverse_proxy in the following
+	// way:
+	//
+	// User Agent ->
+	//  reverse_proxy ->
+	//  forward_proxy_url -> upstream
+	//
+	// Default: off.
```

Comment:
Default is not `off`. It's read from environmental variables.

Reply:
The intent is for the individual module to be able to be configured with a specific proxy. `ForwardProxyURL` takes precedence over `HTTP_PROXY` and no default value is supplied.

Reply:
But `if h.ForwardProxyURL == ""` it still uses `http.ProxyFromEnvironment` therefore the default is still "from environment", not nothing.